src/dispatch/cli.py

```diff
@@ -405,32 +407,41 @@ def upgrade_database(tag, sql, revision, revision_type):
     from .database.manage import init_database
 
     alembic_cfg = AlembicConfig(config.ALEMBIC_INI_PATH)
+    path = None
```

Comment:
It looks like the playwright tests are failing since the `path` is still `None` and line `set_main_option` in line 418 requires a string.